### PR TITLE
Moved figures into figure elements

### PR DESCRIPTION
**Pull request by shannonlucas**
Moved the two figures in the first section into figure elements and added appropriate styling.

Added image of camera lens scaled to desired size.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,35 +21,39 @@ <h1 id="settings">Aperture Settings</h1>
 
   <p class="subheader">Improving your photography by controlling aperture</p>
 
-  <p>Cameras with interchangeable lenses offer three primary settings that affect how a photograph is exposed when the shutter button is pressed. These are aperture, shutter speed, and sensitivity (also called “ISO”) and are often referred to collectively as the “exposure triangle.” Each setting impacts the brightness of a photograph while also providing its own unique impact. This guide will focus on aperture.</p>
```
Comment (ghost) [inline index.html:24]: I put the figure above the first paragraph so that everything flows neatly around it.

File: index.html [hunk 2]
```diff
@@ -70,7 +74,7 @@ <h2 id="priority"><i>Aperture priority</i></h2>
       </tr>
     </thead>
     <tbody>
-      <tr>
```
Comment (ghost) [inline index.html:73]: The only actual change here is that there was an extra space on the end of this line. Those are harmless, but I have my editor set to automatically remove extra spaces at the end whenever it saves.

File: style.css [hunk 3]
```diff
@@ -50,23 +50,55 @@ li a:hover:not(.active) {
   background-color: #04AA6D;
 }
 
-img#lens {
-  max-width: 250px;
-  margin: 0 auto;
+figure {
+  display: table;
```
Comment (ghost) [inline style.css:54]: This display mode on the figure will force the caption text to wrap at the width of the image.

File: style.css [hunk 4]
```diff
@@ -50,23 +50,55 @@ li a:hover:not(.active) {
...
+  width: 100%;
 }
 
 img {
+  display: inline-block;
+  object-fit: contain;
   width: 100%;
   height: 100%;
-  object-fit: contain;
+  max-width: max-content;
```
Comment (ghost) [inline style.css:95]: These max settings cause the images to never get larger than they actually are when the page is resized.

**Comment by smarmo** [comment]
These all look great. I appreciate the feedback and am learning along the way for future projects. Thanks!

**Pull request merged**